### PR TITLE
Adding Docker install

### PR DESCRIPTION
**Pull request by timhallinflux**
Contribution of a Docker Compose based setup for Kapacitor -- and a tutorial to explore TICKscript.

### CODE REVIEW

File: content/kapacitor/v1.3/introduction/install-docker.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
-
 ## Getting Started with TICK and Docker Compose
 
-This short tutorial will demonstrate starting TICK stack components (InfluxDB, Telegraf, Kapacitor) with Docker Compose and then using that stack to learn the rudiments of working with Kapacitor and the [TICKscript](/kapacitor/v1.3/tick/) domain specific language (DSL). It will create a running deployment of these applications that  can be used for an initial evaluation and testing of Kapacitor.  Chronograf is currently not included in the package.
+This short tutorial will demonstrate starting TICK stack components (InfluxDB, Telegraf, Kapacitor) with Docker Compose and then using that stack to learn the rudiments of working with Kapacitor and the [TICKscript](/kapacitor/v1.3/tick/) domain specific language (DSL).  The following discussion is based on the tutorial project package that can be checked out from [Location to be decided](http://github.com). It will create a running deployment of these applications that can be used for an initial evaluation and testing of Kapacitor.  Chronograf is currently not included in the package.
```
Comment (desa) [inline content/kapacitor/v1.3/introduction/install-docker.md]: > [Location to be decided](http://github.com)

Has the location been decided yet?

**Comment by karel-rehor** [comment]
Hi Nate, 

I contributed the install_docker.md.  Your comments left me a bit confused on a first read, since I was trying to understand them in the context of install_docker.  I assume your comments are in the context of the document getting_started.md.  If you need this to be updated I can handle this tomorrow. 

-K

**Comment by nhaugo** [comment]
@karel-rehor yeah, the comments refer to getting started.

**Comment by karel-rehor** [comment]
I've just pushed today's update to getting_started.md to the install-guide branch.  This is not yet ready for review.  I meant to add WIP to the commit comment.  I've replaced a lot of direct addressing of the reader with more passive forms; replaced kapacitor_example db with telegraf db; and added systemctl commands.  I would like to go over this document once more on Monday.

**Comment by karel-rehor** [comment]
I've done some more proofing and style changes.  I also took the time to verify the basic examples on my laptop.  Please review and merge, if everything looks O.K.

**Comment by nhaugo** [comment]
Looks good to me @timhallinflux want to merge?

**Comment by karel-rehor** [comment]
Wait a second.  I'm verifying links, following comments from Ivan.  I noticed, for example that the first link to Influxdb was to v0.9.

**Comment by karel-rehor** [comment]
I've updated some links and then checked all links manually, using markserv.  With relative links I assume that during the publishing process files with .md extensions become the context for the page (e.g. nodes/alert_node.md becomes http://localhost:8000/kapacitor/v1.3/nodes/alert_node/) or if the link is to a directory, the page context will refer to the file index.md.

**Comment by ivankudibal** [comment]
links fixed -- this can be merged

**Pull request merged**